x-pack/legacy/plugins/siem/public/components/matrix_histogram/utils.test.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  getBarchartConfigs,
+  DEFAULT_CHART_HEIGHT,
+  DEFAULT_Y_TICK_FORMATTER,
+  formatToChartDataItem,
+  getCustomChartData,
+} from './utils';
+import { UpdateDateRange } from '../charts/common';
+import { Position, ScaleType, TickFormatter } from '@elastic/charts';
+import { MatrixOverTimeHistogramData } from '../../graphql/types';
+
+describe('utils', () => {
+  describe('getBarchartConfigs', () => {
+    interface BarchartConfigs {
```

Comment:
Thanks for defining this new interface! Consider moving it from `utils.test.ts` to `utils.ts`, and updating the type definition of `getBarchartConfigs` to return the `BarchartConfigs` interface   i.e.:

```
export const getBarchartConfigs = ({
  chartHeight,
  from,
  legendPosition,
  to,
  onBrushEnd,
  yTickFormatter,
  showLegend,
}: GetBarchartConfigsProps): BarchartConfigs => ({
```